modules/face/data/mean_shape.txt

```diff
@@ -0,0 +1,195 @@
+137.856,219.266
```

Comment:
It would be nice to store the landmark locations procentually to the size of the face bounding box. With absolute values I am afraid the system might be easily broken down with slightly different detector outputs. Is that something you can change?